packages/platform-feature-tests/cypress/page-objects/base-page.js

```diff
@@ -0,0 +1,43 @@
+export class BasePage {
+
+    elements = {
```

Comment:
This object will be overridden in the subpage classes, so I would suggest giving it a name like `basePageElements`. Or you could have a constructor in you subclasses and just add to the elements object, for example
`
class SubPage extends BasePage {
  constructor() {
    super()
    this.element.newElement = '[data-cy="..."]'
  }
}
`